build.gradle

```diff
@@ -84,6 +86,72 @@ check.doLast {
 }
 /////////////////// END jmh /////////////////////
 
+/////////////////////////////////////////////////////////////////
+//                   SonaType Central Repo                     //
+/////////////////////////////////////////////////////////////////
+/*
+task javadocJar(type: Jar) {
+    classifier = 'javadoc'
+    from javadoc
+}
+*/
+
+task sourcesJar(type: Jar) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+artifacts {
+    archives sourcesJar
+}
+
+signing {
+    sign configurations.archives
```

Comment:
Been awhile since I've used gradle. Isn't there supposed to be a `gradle.properties` or a `build.properties` file somewhere that would contain signing credentials?